packages/frontend/src/hooks/dashboard/useDashboard.tsx

```diff
@@ -210,6 +220,30 @@ export const useCreateMutation = (
     );
 };
 
+export const useDuplicateDashboardMutation = (id: string) => {
```

Comment:
what's the main purpose of these mutations ? why don't we make a call to `duplicateDashboard` method directly ?

Reply:
Because i was doing all the duplicating object logic here, but yes now its useless :) TY